Automattic/wp-calypso
# Nav Unification: Fix modal images on Safari and include accounts link

#### Changes proposed in this Pull Request

* Specify `align-items: flex-end` for the sake of Safari as their default is `stretch` for the same property.
* Add account settings link to last page of the modal
* Replace JPEG imges with PNG images with transparent background to avoid color degradation from compression.
* Moved image from below the text to above it RE: https://github.com/Automattic/wp-calypso/pull/49935#issuecomment-776813639


#### Testing instructions

* Visit the Calypso.live link on Safari (both on mobile and desktop if possible)
* Remove your user preference via the debug badge (bottom right) for `nav-unification-modal`
* Observe the image aspect ratio in Safari
* Check the images don't contain two shades of the blue background
* Check the last slide has a link to account settings

<table>
<tr>
	<td>Before</td>
	<td>After</td>
</tr>
<tr>
	<td>
		<img src="https://user-images.githubusercontent.com/8639742/107505109-21b87280-6b94-11eb-93ed-fa00f0cfde81.png" />
	</td>
	<td>
	<img src="https://user-images.githubusercontent.com/8639742/107523286-7a940500-6bac-11eb-899e-040cfcea9190.png" />
	</td>
</tr>
<tr>
	<td><img src="https://user-images.githubusercontent.com/8639742/107055202-f23aec00-67c8-11eb-8268-21c9cdc1b742.png" /></td>
	<td><img src="https://user-images.githubusercontent.com/8639742/107619228-c4332d00-6c4a-11eb-9f9d-49b1fccc9b18.png" /></td>
</tr>
</table>

Fixes https://github.com/Automattic/wp-calypso/issues/49770
